src/main/java/uk/gov/hmcts/reform/cwrdapi/servicebus/TopicPublisher.java

```diff
@@ -8,28 +8,41 @@
 import org.springframework.retry.annotation.Recover;
 import org.springframework.retry.annotation.Retryable;
 import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.cwrdapi.client.domain.PublishCaseWorkerData;
 import uk.gov.hmcts.reform.cwrdapi.controllers.advice.CaseworkerMessageFailedException;
+import uk.gov.hmcts.reform.cwrdapi.service.IValidationService;
+
+import javax.validation.constraints.NotNull;
 
 @Slf4j
 @Service
 public class TopicPublisher {
 
     private final JmsTemplate jmsTemplate;
     private final String destination;
+    private final IValidationService validationService;
 
     @Value("${loggingComponentName}")
     private String loggingComponentName;
 
     @Autowired
     public TopicPublisher(JmsTemplate jmsTemplate,
-                          @Value("${crd.publisher.azure.service.bus.topic}") final String destination) {
+                          @Value("${crd.publisher.azure.service.bus.topic}") final String destination,
+                          IValidationService validationService) {
         this.jmsTemplate = jmsTemplate;
         this.destination = destination;
+        this.validationService = validationService;
     }
 
-    @Retryable(maxAttempts = 5, backoff = @Backoff(delay = 2000, multiplier = 3))
-    public void sendMessage(Object message) {
+    @Retryable(maxAttempts = 3, backoff = @Backoff(delay = 2000, multiplier = 3))
+    public void sendMessage(@NotNull Object message) {
         log.info("{}:: Publishing message to service bus topic.", loggingComponentName);
+        if (message instanceof PublishCaseWorkerData) {
+            log.info("Job Id is: {}, Count of User Ids is: {} ", validationService.getJobId(),
```

Comment:
Can we add the loggingComponent name in the log please ?

Reply:
Sure, thanks for the catch :)

Reply:
Updated the code and added logging component name.

Reply:
thanks :)